<?xml version="1.0"?>
<launch>
    <!-- controller lib -->
    <node pkg="underwater_robot" type="joystick_control" name="joystick_control"/>	

</launch>
